types/v3/tech-record/get/car/complete/index.d.ts

```diff
@@ -51,22 +51,22 @@ export interface TechRecordGETCarComplete {
   techRecord_applicantDetails_emailAddress?: null | string;
   createdTimestamp: string;
   partialVin?: null | string;
-  primaryVrm?: null | string;
+  primaryVrm: null | string;
```

Comment:
How come some of the properties have had the null operator removed, but not others? 🙂

Reply:
Good spot, should remove the `null` value.

Reply:
Will the json definition also need updating?

Reply:
Types are generated from the json definitions, so only need to update those.